### ISSUE TITLE
Unit Tested Failed for Module 03 in Tinytorch

### ISSUE THREAD

**Issue by Takosaga** [harvard-edge/cs249r_book#1109]
Using notebook created from  `tito module start 03`, the tests fail in the notebook due to `requires_grad=True` flag for the weight and bias even though it states the flag marks these tensors for gradient computation in Module 06. Though I remove in the notebook to pass the test, `tito module complete 03` has ❌ Unit tests failed at `test_unit_linear_layer()`

I used `tito module reset 03` to reset notebook and still get same errors.
